bin/hdfcoinc/pycbc_coinc_mergetrigs

```diff
@@ -86,8 +85,8 @@ for filename in args.trigger_files:
         logging.error('Cannot open %s', filename)
         raise e
     ifo_data = data[ifo]
-    s, e = ifo_data['search/start_time'][:], ifo_data['search/end_time'][:]
-    start, end = numpy.append(start, s), numpy.append(end, e)
+    segments.add((float(ifo_data['search/start_time'][:]),
```

Comment:
looks OK. I would just put a comment eg 'Don't add segments that duplicate existing ones'